dashboard/frontend/styles/_basics.scss

```diff
@@ -0,0 +1,10 @@
+* {
+    margin: 0;
```

Comment:
You can also use `tailwind` classnames for this to keep it consistent.
Ref: https://tailwindcss.com/docs/using-with-preprocessors